### ISSUE TITLE
New `display` library

### ISSUE THREAD

**Issue by siliconwitch** [brilliantlabsAR/monocle-micropython#151]

**Comment by josuah** [comment]
Not sure how to use the FBGraphics block yet. This one has the SPI API for sending graphical objects.
See https://github.com/brilliantlabsAR/monocle-fpga/issues/9

**Comment by josuah** [comment]
https://github.com/brilliantlabsAR/monocle-micropython/pull/159/files

**Comment by josuah** [comment]
Currently blocked by the bitstream transfer time on every startup.

**Comment by josuah** [comment]
Currently blocked by the choice for a new API for the display module...
Or mabye nothing at first, letting user decide?

**Comment by josuah** [comment]
- [ ] document the API
- [ ] implement the API as shown below
- [ ] flip the display vertically and horizontally (180° spin)
- [ ] check the coordinate system for both text so that (0, 0) is at the bottom left

API:

```python
display.Line()
display.Rectangle()
display.Polygon()
display.Polyline()
display.Text()

.move()

display.show()
```

Usage:

```python
l1 = display.Line()
display.show(l1)

l2 = display.Line()
p1 = display.Polygon()
grp = [l1, l2, p1]
display.show(grp)

l1.move()
grp.move()
```

**Comment by josuah** [comment]
Fixed by #173 (WIP)
